utils.py

```diff
@@ -65,7 +65,7 @@ def populate_parser(parser):
     parser.add_argument("-t", "--session-timeout", required=False,
                         help="Session expiration timeout in seconds.")
     parser.add_argument("--force", required=False, default=False, action="store_true",
-                        help="Force run the command with no confirmation.")
+                        help="Force run the command with no confirmation. WARNING! - this will set unsafe-auto-accept to be true as well.")

```

Comment:
You need to add logic to override unsafe-auto-accept flag before initialize APIClientArgs